### PR TITLE
chore(upgrade): Upgraded for 2.1.

### PR DESCRIPTION
**Pull request by brettp**

### CODE REVIEW

File: actions/apiadmin/regenerate.php [hunk 1]
```diff
@@ -2,39 +2,41 @@
...
-*/
+ * @link      http://www.elgg.org
+ */
 
-global $CONFIG;
+$key = (int)get_input('keyid');
+$obj = get_entity($key);
 
-admin_gatekeeper();
+if (!($obj instanceof \ElggEntity) || !$obj->getSubtype() === 'api_key') {
```
Comment (mrclay) [inline actions/apiadmin/regenerate.php]: `$obj->getSubtype() !== 'api_key'`

File: actions/apiadmin/renamekey.php [hunk 2]
```diff
@@ -2,29 +2,27 @@
...
 $obj = get_entity($key);
 
-if ( $obj && ($obj instanceof ElggObject) && ($obj->subtype == get_subtype_id('object', 'api_key')) ) {
-	$obj->title = $ref;
-	if ( $obj->save() ) {
-		system_message(elgg_echo('apiadmin:refrenamed'));
-	} else {
-		register_error(elgg_echo('apiadmin:refnotrenamed'));
-	}
+if (!($obj instanceof \ElggEntity) || !$obj->getSubtype() === 'api_key') {
```
Comment (hypeJunction) [inline actions/apiadmin/renamekey.php]: !==

File: views/default/object/api_key.php [hunk 3]
```diff
@@ -3,70 +3,78 @@
...
-$info .= "</p></div>";
-$info .= "<div><p><b>$public_label:</b> {$entity->public}<br />";
+
+// revoke
+$item = ElggMenuItem::factory([
+	'name' => "revoke",
+	'text' => elgg_view_icon('delete'),
+	'title' => elgg_echo("apiadmin:revoke"),
+	'confirm' => elgg_echo("apiadmin:revoke_prompt"),
+	'href' => elgg_http_add_url_query_elements("action/apiadmin/revokekey", ['keyid' => $entity->getGUID()]),
```
Comment (mrclay) [inline views/default/object/api_key.php:26]: How do these URLs get tokens?

File: actions/apiadmin/revokekey.php [hunk 4]
```diff
@@ -2,27 +2,25 @@
...
-
 $obj = get_entity($key);
 
-if ( $obj && ($obj instanceof ElggObject) && ($obj->subtype == get_subtype_id('object', 'api_key')) ) {
-	if ( $obj->delete() ) {
-		system_message(elgg_echo('apiadmin:keyrevoked'));
-	} else {
-		register_error(elgg_echo('apiadmin:keynotrevoked'));
-	}
+if (!($obj instanceof \ElggEntity) || !$obj->getSubtype() === 'api_key') {
```
Comment (hypeJunction) [inline actions/apiadmin/revokekey.php]: !==
I tend to use elgg_instanceof for subtype comparison

File: views/default/object/api_key.php (hunk 3 above)
Comment (hypeJunction) [inline views/default/object/api_key.php:26]: Confirm sets tokens

File: views/default/object/api_key.php [hunk 5]
```diff
@@ -3,70 +3,78 @@
...
+	'is_trusted' => true,
```
Comment (hypeJunction) [inline views/default/object/api_key.php]: All local URLs are trusted by default

**Comment by mrclay** [comment]
manifest should require web_services plugin.

File: actions/apiadmin/revokekey.php (hunk 4 above)
Comment (brettp) [inline actions/apiadmin/revokekey.php]: Oh I forgot about that function!

**Comment by hypeJunction** [comment]
Some issues with indentation, but nothing critical. LGTM

**Comment by brettp** [comment]
Where? Might as well take care of everything at once.

**Pull request closed without merging**